tests/udpecho.py

```diff
@@ -0,0 +1,10 @@
+import socket
```

Comment:
Script seems useful, but perhaps should be in a `scripts` dir, or even `contrib` sub-package?